bleak/backends/bluezdbus/manager.py

```diff
@@ -917,6 +936,9 @@ def _parse_msg(self, message: Message):
                 except KeyError:
                     pass
 
+                if obj_path in self._properties and not self._properties[obj_path]:
```

Comment:
I think this should be after the for loop so that it only runs once after all interfaces have been removed. A comment would be nice too since it isn't super-obvious why we are doing this.

Reply:
Also wouldn't hurt to split this one out into a separate commit since it isn't strictly related to the other changes.